### PR TITLE
Added title to the download and github icons like darkMode button

### PR DESCRIPTION
**Pull request by Hrshk15**
Earlier when user hovered over the download and github buttons nothing used to show in order to tell user what these buttons will do like the darkmode button used to tell.

Now I have added the titles to download and github buttons, when user hover over download button it shows - "download sbuttons.min.css" and when user hovers over github button it shows "sbuttons repo".
Issue:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -111,15 +111,16 @@
...
+>
             <i class="fas fa-download"></i>
           </a>
           <a
             href="https://github.com/sButtons/sbuttons"
             target="_blank"
             class="icon-link"
           >
-            <i class="fab fa-github"></i>
+            <i class="fab fa-github" title="sbuttons repo"></i>
```
Comment (shahednasser) [inline index.html:123]: Change sbuttons to "sButtons"

**Review by shahednasser — CHANGES_REQUESTED** [review]
Great @Hrshk15 but this change will only be seen in the homepage. Can you also add it in the documentation and examples pages?

**Comment by Hrshk15** [comment]
@shahednasser  Done with the required changes

**Review by Hrshk15 — COMMENTED** [review]
checked

**Review by shahednasser — CHANGES_REQUESTED** [review]
Great but please remove all changes in `dist/sbuttons.css` as they are unnecessary

**Comment by Hrshk15** [comment]
> Great but please remove all changes in `dist/sbuttons.css` as they are unnecessary

I am not able to get it, I have not even edited anything in dist/sbuttons.css .

**Comment by shahednasser** [comment]
@Hrshk15 the easiest way is to just copy the content of the file in the repo then committing the changes.

**Comment by shahednasser** [comment]
@Hrshk15 any update on this?

**Pull request closed without merging**